www/docs/en/dev/guide/platforms/android/webview.md

```diff
@@ -38,97 +38,56 @@ legacy `CordovaActivity` component that pre-dates the 1.9 release.
-   the referenced view id matches the `id` attribute specified in the
-   XML fragment shown above:
-
-        public class CordovaViewTestActivity extends Activity implements CordovaInterface {
-            CordovaWebView cwv;
-            /* Called when the activity is first created. */
-            @Override
-            public void onCreate(Bundle savedInstanceState) {
-                super.onCreate(savedInstanceState);
-                setContentView(R.layout.main);
-                cwv = (CordovaWebView) findViewById(R.id.tutorialView);
-                Config.init(this);
-                cwv.loadUrl(Config.getStartUrl());
-            }
-
-1. If the application needs to use the camera, implement the
-   following:
-
-        @Override
-        public void setActivityResultCallback(CordovaPlugin plugin) {
-            this.activityResultCallback = plugin;
-        }
-        /**
-         * Launch an activity for which you would like a result when it finished. When this activity exits,
-         * your onActivityResult() method is called.
-         *
-         * @param command           The command object
-         * @param intent            The intent to start
-         * @param requestCode       The request code that is passed to callback to identify the activity
-         */
-        public void startActivityForResult(CordovaPlugin command, Intent intent, int requestCode) {
-            this.activityResultCallback = command;
-            this.activityResultKeepRunning = this.keepRunning;
-
-            // If multitasking turned on, then disable it for activities that return results
-            if (command != null) {
-                this.keepRunning = false;
-            }
-
-            // Start activity
-            super.startActivityForResult(intent, requestCode);
-        }
-
-        @Override
-        /**
-         * Called when an activity you launched exits, giving you the requestCode you started it with,
-         * the resultCode it returned, and any additional data from it.
-         *
-         * @param requestCode       The request code originally supplied to startActivityForResult(),
-         *                          allowing you to identify who this result came from.
-         * @param resultCode        The integer result code returned by the child activity through its setResult().
-         * @param data              An Intent, which can return result data to the caller (various data can be attached to Intent "extras").
-         */
-        protected void onActivityResult(int requestCode, int resultCode, Intent intent) {
-            super.onActivityResult(requestCode, resultCode, intent);
-            CordovaPlugin callback = this.activityResultCallback;
-            if (callback != null) {
-                callback.onActivityResult(requestCode, resultCode, intent);
-            }
-        }
-
-1. Finally, remember to add the thread pool, otherwise plugins
```

Comment:
What about this?

Reply:
Again, not sure if this still applicable. I can look into this when I work on the plugin for the native-to-cordova-bridge